cadcutils/cadcutils/net/tests/test_auth.py

```diff
@@ -159,9 +159,9 @@ def test_get_cert_main_help(self):
         """ Test the help option of the cadc-get-cert app """
```

Comment:
Hard to tell. It's good if you mistakenly change the help text without realizing. It could also pick up differences due to changes when upgrading argparse. Also good for coverage. It's the equivalent of testing a GUI.